src/blocks/_nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+  flex-shrink: 0;
```

Comment:
that doesn't make sense here, flex-shrink only work for flex children, your block shouldn't be dependant on outer block
```suggestion
```

Reply:
Completely agree - thats my redunt code after experementing with various conditions . 
Aprreciate  you mentioning.